spec/values/calculation/round/one_argument.hrx

```diff
@@ -0,0 +1,200 @@
+}
+
+<===> interpolation/output.css
+a {
+  e: round(3px + 3);
+}
+
+<===>
+================================================================================
+<===> math/slash_as_division/input.scss
+b {
+  a: 2px / round(1.5);
+}
+
+<===> math/slash_as_division/output.css
+b {
+  a: 1px;
+}
+
+<===> math/slash_as_division/warning
+DEPRECATION WARNING: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.
+Recommendation: math.div(2px, round(1.5)) or calc(2px / round(1.5))
+More info and automated migrator: https://sass-lang.com/d/slash-div
+  ,
+2 |   a: 2px / round(1.5);
+  |      ^^^^^^^^^^^^^^^^
+  '
+    input.scss 2:6  root stylesheet
+
+<===>
+================================================================================
+<===> error/named_argument/variable/input.scss
+a {b: round($number: var(--c))}
+
+<===> error/named_argument/variable/error
+Error: $number: var(--c) is not a number.
+  ,
+1 | a {b: round($number: var(--c))}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/unsimplifiable/input.scss
+a {b: round(17px + 5%)}
+
+<===> error/unsimplifiable/error
+Error: Single argument 17px + 5% expected to be simplifiable.
+  ,
+1 | a {b: round(17px + 5%)}
+  |       ^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/variables/input.scss
+a {
+  b: round(1px + 2px - var(--c))
+}
+
+<===> error/variables/error
+Error: Single argument 3px - var(--c) expected to be simplifiable.
+  ,
+2 |   b: round(1px + 2px - var(--c))
+  |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 2:6  root stylesheet
+
+<===>
+================================================================================
+<===> error/type/input.scss
+a {b: round("0")}
+
+<===> error/type/error
+Error: $number: "0" is not a number.
+  ,
+1 | a {b: round("0")}
+  |       ^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/no_args/input.scss
+a {b: round()}
+
+<===> error/no_args/error
+Error: Missing argument $number.
+  ,--> input.scss
+1 | a {b: round()}
+  |       ^^^^^^^ invocation
+  '
+  ,--> sass:math
+1 | @function round($number) {
+  |           ============== declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/too_many_args/input.scss
+a {b: round(1, 2, 3, 4)}
+
+<===> error/too_many_args/error
+Error: Only 1 argument allowed, but 4 were passed.
+  ,--> input.scss
+1 | a {b: round(1, 2, 3, 4)}
+  |       ^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:math
+1 | @function round($number) {
+  |           ============== declaration
+  '
+  input.scss 1:7  root stylesheet
```

Comment:
These tests aren't specific to the single-argument round function, so they should probably be in a different directory.

Reply:
Does it make more sense to move it to the three-argument round tests?